dask/array/core.py

```diff
@@ -3315,7 +3319,8 @@ def to_zarr(
 
     See Also
     --------
-    dask.array.Array.compute_chunk_sizes
+    :func:`dask.array.store`
+    :meth:`dask.array.Array.compute_chunk_sizes`
```

Comment:
Hmm I wonder if `:func:` and `:meth:` are needed in "See Also" sections? Looking at the [current `to_zarr` docstring](https://docs.dask.org/en/latest/array-api.html#dask.array.to_zarr) it appears `dask.array.Array.compute_chunk_sizes` is already a hyperlink to the corresponding docstring

Reply:
They are not required (numpydoc handles it automatically).

Reply:
Nice, good to know! @djhoese would you mind removing the extra `:func:` and `:meth:` parts? After that, this should be good to merge

Reply:
Thanks. That was left over when I was going to try to see if it made sense to shorten the link text, but then changed my mind. This should not be fixed.